source/jobs/designer.erb

```diff
@@ -142,7 +142,7 @@ related:
 
   <%= partial 'jobs/quote_caricature', locals: { section: current_page.data.design_quote } %>
 
-  <%= partial 'jobs/contact_us' %>
+  <%= partial 'jobs/contact_us', locals: {roles: 1} %>
```

Comment:
I think it might be a small improvement to use 1 as the default for the number of roles now I think about it a bit more. Then only the call to render from the jobs landing page requires this extra argument.

```
<% roles ||= 1 %>
```

at the top of the partial should do the trick. Then we don't need to pass it in explicitly from the individual role pages.